dd-library-php-setup.php

```diff
@@ -892,4 +1252,16 @@ function get_supported_php_versions()
     return ['5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1'];
 }
 
+function delete_on_exit($dir)
```

Comment:
I really like this idea. Can we restrict allowed paths to the prefix `/tmp/`? Just to make sure we are not causing major issues with a bug in our installer, since it will have to be run as root.
Moreover please add a docblock specifying this restriction.

In a follow up PR, I can then make sure this function is used all over the places (even in the tracer installer).